src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -70,7 +78,41 @@ export default defineComponent({
 
     currentSortValue: function () {
       return (this.sortNewest) ? 'newest' : 'top'
-    }
+    },
+
+    observeVisibilityOptions: function() {
+      if (!this.commentAutoLoadEnabled) { return false }
+      if (!this.videoPlayerReady) { return false }
+
+      return {
+        callback: (isVisible, _entry) => {
+          // This is also fired when **hidden**
+          // No point doing anything if not visible
+          if (!isVisible) { return }
+          // It's possible the comments are being loaded/already loaded
+          if (this.canPerformInitialCommentLoading) {
+            this.getCommentData()
+          }
+          if (this.canPerformMoreCommentLoading) {
+            this.getMoreComments()
+          }
```

Comment:
Could you please make the second if an else if, so that it is more obvious to someone reading the code, that only one of them will get triggered at a given time.